### PR TITLE
phi-2: fix --temp/--seed arguments.

### PR DESCRIPTION
**Pull request by idoru**
`seed` was not being passed to generate(). 

This quick fix will let folks experiment with `--temp/--seed`!

### CODE REVIEW

**Review by awni — APPROVED** [review]
Thanks, that was on oversight. You wrote something about fixing `--seed` is that supposed to be in this PR?

**Comment by idoru** [comment]
Sorry if it was a bit confusing. I just meant that `--seed` did not have any effect until this fix because when the temperature is defaulted to 0 the `logits` are sampled with `argmax()` and not `random()`
